src/core_mapfiles/std.rs

```diff
@@ -22,9 +22,9 @@ static STD_06: &CoreSignatures = &CoreSignatures {
         (Th06, 0, Some(("fff", None))),
         (Th06, 1, Some(("Cff", None))),
         (Th06, 2, Some(("fff", None))),
-        (Th06, 3, Some(("S__", None))),
-        (Th06, 4, Some(("S__", None))),
-        (Th06, 5, Some(("___", None))),
+        (Th06, 3, Some(("S________", None))),
+        (Th06, 4, Some(("S________", None))),
+        (Th06, 5, Some(("____________", None))),
```

Comment:
I'm not sure I like this, why did we decide against having `_` align to the next dword?  `-` could be the single byte, as you were using earlier

Reply:
I know I'd been suggesting making `_` align to dword, but I'm starting to see why you were resistant to the idea. If anything it'd probably be best to just treat `_` as a forced 4 bytes of padding and `-` as 1.

Reply:
I don't remember being resistant to making it align to dword boundary.  However, maybe keeping it 4 is simpler